CONTRIBUTING.md

```diff
@@ -1,70 +1,105 @@
-# Contribution Guidelines
+Contributing to Shields
+=======================
 
-This is the home of Shields.io, home to the badge design specification, API documentation, and server code for Badges as a Service.
+Shields is a community project! We invite your participation through issues
+and pull requests.
 
-We invite participation through [GitHub Issues][], which we use much like a discussion forum. This repository should only contain non-implementation specific topics: specifications, design, and the web site.
 
-## This implementation
+Ways you can help
+-----------------
 
-Please see [INSTALL.md][] for information on how to start contributing code to
-shields.io.
-You can read a [Tutorial on how to add a badge](doc/TUTORIAL.md).
+### Contributing code
 
-[INSTALL.md]: ./INSTALL.md
+This project has quite a backlog of suggestions! If you're new to the project,
+maybe you'd like to opan a pull request to address one of them:
 
-Note that the root gets redirected to <http://shields.io>.
-For testing purposes, you can go to `http://localhost/try.html`.
+[![GitHub issues by-label](https://img.shields.io/github/issues/badges/shields/beginner-friendly.svg)]()
```

Comment:
This could link to https://github.com/badges/shields/issues?q=is%3Aissue+is%3Aopen+label%3Abeginner-friendly. What do you think about it?

Reply:
Great idea! Updated.